packages/Webkul/Shop/src/Resources/views/components/layouts/header/mobile/index.blade.php

```diff
@@ -210,12 +258,12 @@ class="px-5 py-2 text-[16px] hover:bg-gray-100 cursor-pointer"
                 class="icon-search flex items-center absolute left-[12px] top-[12px] text-[25px] pointer-events-none">
             </div>
 
-            <input 
+            <input
                 type="text"
                 class="block w-full px-11 py-3.5 border border-['#E3E3E3'] rounded-xl text-gray-900 text-xs font-medium"
                 name="query"
                 value="{{ request('query') }}"
-                placeholder="Search for products" 
+                placeholder="Search for products"
```

Comment:
Hi, can you please add the translation also? I know its already there. It will be good if you add it.

Reply:
Okay Sure